clickhouse_sqlalchemy/drivers/base.py

```diff
@@ -466,13 +508,48 @@ def _get_col_type(self, name, spec):
                      (spec, name))
                 return sqltypes.NullType
 
-    def _get_column_info(self, name, format_type):
-        return {
-            'name': name,
-            'type': self._get_col_type(name, format_type),
-            'nullable': True,
-            'default': None,
-        }
+    @staticmethod
+    def _parse_enum_options(option_string):
```

Comment:
Let's generalize this method with the final version of `_parse_options`: https://github.com/mymarilyn/clickhouse-driver/pull/67.